docs/content/standards/kiosk.mdx

```diff
@@ -404,7 +404,7 @@ public fun mutable_borrow_example<T>(
 
 ### Mutable borrow with borrow_val
 
-You can use the PTB-friendly kiosk::borrow_val function. It allows you to take an asset and place it back in the same transaction. To make sure the asset is placed back into the kiosk, the function "obliges" the caller with a “Hot Potato”.
+You can use the PTB-friendly kiosk::borrow_val function. It allows you to take an asset and place it back in the same transaction. To make sure the asset is placed back into the kiosk, the function obliges the caller with a “Hot Potato”.
```

Comment:
```suggestion
Use the PTB-friendly `kiosk::borrow_val` function to take an asset and place it back in the same transaction. To make sure the asset is placed back into the kiosk, the function obliges the caller with a Hot Potato. See The Move Book for more information on the [Hot Potato pattern](https://move-book.com/programmability/hot-potato-pattern).
```
Prefer this over making Hot Potato in the original text the hyperlink because it lets the reader know they are leaving this site.